gateway-js/src/index.ts

```diff
@@ -1008,9 +972,6 @@ export class ApolloGateway implements GraphQLService {
     switch (this.state.phase) {
       case 'initialized':
       case 'failed to load':
-        throw Error(
```

Comment:
**note** It's safe to call `gateway.stop`, even if the gateway is only `initialized` or `failed to load`. There's no public interface to know if the gateway is in a good state to stop anyway.